code/game/objects/items/miningradio.dm

```diff
@@ -0,0 +1,176 @@
+#define WEATHER_ALERT_CLEAR 0
+#define WEATHER_ALERT_INCOMING 1
+#define WEATHER_ALERT_IMMINENT_OR_ACTIVE 2
+
+/// Portable mining radio purchasable by miners
+/obj/item/radio/weather_monitor
+	icon = 'icons/obj/miningradio.dmi'
+	name = "mining weather radio"
+	icon_state = "miningradio"
+	desc = "A weather radio designed for use in inhospitable environments. Gives audible warnings when storms approach. Has access to cargo channel."
+	freqlock = RADIO_FREQENCY_LOCKED
+
+/obj/item/radio/weather_monitor/Initialize(mapload)
+	. = ..()
+	AddComponent( \
+		/datum/component/weather_announcer, \
+		state_normal = "weatherwarning", \
+		state_warning = "urgentwarning", \
+		state_danger = "direwarning", \
+	)
+	set_frequency(FREQ_SUPPLY)
+
+/datum/orderable_item/mining/weather_radio
+	item_path = /obj/item/radio/weather_monitor
+	cost_per_order = 500
```

Comment:
```suggestion
	cost_per_order = 250
```

No need to make every single item expensive, let's just make some things nice to have, eh? Also doesn't this need a category to be set or something?

Reply:
Also, it'd be neat if this thing also had information on the gas makeup of lavaland. Just a thought.

Reply:
> Also, it'd be neat if this thing also had information on the gas makeup of lavaland. Just a thought.

there is a gas analyzer in the toolbox on lavaland that does this already, and i don't think them having 1:1 overlap is super interesting. the price is subject to change tho

Reply:
price has been lowered from 500 -> 400
this effectively means at roundstart it will cost as much as stabilizing serum rather than explorer's webbing (around 260 points last i checked).